pageserver/src/tenant/storage_layer/delta_layer.rs

```diff
@@ -1050,10 +1028,10 @@ impl<T: AsRef<DeltaLayerInner>> ValueRef<T> {
     }
 }
 
-struct Adapter<T: AsRef<DeltaLayerInner>>(T);
+pub(crate) struct Adapter<T: AsRef<DeltaLayerInner>>(T);
```

Comment:
This bound should be possible to get rid of. Though, I did try that unsure why it wasn't possible.